app/Listeners/SendTaggedNotification.php

```diff
@@ -2,8 +2,8 @@
 
 namespace Rogue\Listeners;
 
-use Notification;
 use Rogue\Events\PostTagged;
+use Illuminate\Support\Facades\Notification;
```

Comment:
This seemed weird to me. Not sure why the code in the event listener worked find even though the `use` statement was borked, referring to a non-existent, global `Notification` class 🤷‍♂️

Reply:
This is one of those very spooky things about facades! Laravel actually intercepts the "class not found" event emitted by PHP and subs in the facade, so I think(?!) excluding the import altogether is the intended usage. (But personally, I find that super confusing so glad we added this import in!)